src/framework/uima/flow_controller.hpp

```diff
@@ -0,0 +1,162 @@
+#ifndef UIMA_FLOW_CONTROLLER_HPP
+#define UIMA_FLOW_CONTROLLER_HPP
+
+/** \file flow_controller.hpp .
+-----------------------------------------------------------------------------
+
+
+
+
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+
+-----------------------------------------------------------------------------
+
+   Description:
```

Comment:
Please add a description, just grab the start of the description in the code below.

Some of the framework files have entries describing when they were modified. I find that useful to gauge code maturity.

Please add a section here after description, separated with '------------------------'  that contains the current date and "Initial creation". Take a look at `src/framework/uima/annotator_mgr.hpp` for an example.